src/deepqmc/train.py

```diff
@@ -187,6 +187,9 @@ def train(  # noqa: C901
                     }
                     if workdir:
                         update_tensorboard_writer(writer, step, pretrain_stats)
+            smpl_state = sampler.init(
+                rng, partial(ansatz.apply, params), {}, sample_size, state_callback
```

Comment:
Wasn't sure what to put in for `wf_state` as there isn't one in scope. This seemed to work

Reply:
Passing an empty state is fine here, as the `wf_state` gets updated in the initialization of the sampler.

Reply:
Passing `{}` as `wf_state` to `sampler.init` seems to be a somewhat common pattern (happens in `pretrain.py` as well), so we should probably make `wf_state` an optional argument. See [814d691](https://github.com/deepqmc/deepqmc/pull/72/commits/814d69196c724ce3898d4d5026181451f06759aa).